scc/tests/args.rs

```diff
@@ -100,7 +100,7 @@ fn file_directory_orders(
 }
 
 #[cfg(test)]
-mod tests {
+mod args {
```

Comment:
this is now a nested module `args::args`
maybe the module should be removed and the tests made global?
that's the convention for integration tests like this: https://doc.rust-lang.org/book/ch11-03-test-organization.html#integration-tests

Reply:
Ah, gotcha. Will do!